internal/runbits/checkout/checkout.go

```diff
@@ -55,10 +54,10 @@ func (e errCommitDoesNotBelong) Error() string {
 var errNoCommitID = errs.New("commitID is nil")
 
 func New(repo git.Repository, prime primeable) *Checkout {
-	return &Checkout{repo, prime.Output(), prime.Config(), prime.Analytics(), "", prime.Auth()}
+	return &Checkout{repo, prime.Output(), prime.Config(), prime.Analytics(), prime.Auth()}
 }
 
-func (r *Checkout) Run(ns *project.Namespaced, branchName, cachePath, targetPath string, noClone, fromArchive bool) (_ string, rerr error) {
+func (r *Checkout) Run(ns *project.Namespaced, branchName, cachePath, targetPath string, noClone, onlyValidate bool) (_ string, rerr error) {
```

Comment:
`onlyValidate` doesn't seem an appropriate name to me. If supplied it in fact does no validations, and it only does the checkout (ie. create the project files).

Perhaps `bareCheckout` would be more appropriate?

Reply:
Okay.